packages/transport/src/core/CoreErrors.ts

```diff
@@ -14,6 +14,20 @@ class Relationships {
         );
     }
 
+    public alreadyExists(status: RelationshipStatus) {
+        return new CoreError(
+            "error.transport.relationships.alreadyExists",
+            `No new Relationship to the peer can be created as a Relationship in status '${status}' currently exists.`
+        );
+    }
+
+    public notYetDecomposedByPeer() {
+        return new CoreError(
+            "error.transport.relationships.notYetDecomposedByPeer",
+            "No new Relationship can be created as the former Relationship is not yet fully decomposed by the peer."
```

Comment:
I'd omit the "fully", since the peer may either have called the decompose route or not but nothing in between.